lib/errors/no-ref-image-error.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var inherit = require('inherit'),
+    StateError = require('./state-error');
+
+var NoRefImageError = inherit(StateError, {
+    /**
+     * @constructor
+     * @param {String} refImagePath
+     * @param {StateResult} result
+     */
+    __constructor: function(refImagePath, result) {
+        this.message = 'Can not find reference image at ' + refImagePath + '\n' +
+            'Run `gemini gather` command to capture all reference images.';
+
+        this.suiteName = result.suiteName;
```

Comment:
@hartoman, no reason to do it, you are right. I though `StateError` already does all all assigments, but it turned out not to be true.